include/vrv/object.h

```diff
@@ -905,7 +905,7 @@ class Object : public BoundingBox {
     /**
      * Calculate the slur direction
      */
-    virtual int PrepareSlurs(FunctorParams *) { return FUNCTOR_CONTINUE; }
+    virtual int CalcSlurs(FunctorParams *) { return FUNCTOR_CONTINUE; }
```

Comment:
Can we rename this into `CalcSlurDirection`? That's all it does for now. The actual slur calculation happens during drawing and in the `AdjustSlurs` functor.